recipes/compyle/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "compyle" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  path: "./pypr-compyle-v0.5-112-g42a7868.tar.gz"
```

The source needs to be downloaded from somewhere. Can you force push to this branch to remove the binary from the history? It's not too bad as the tarball is small but it's still a good idea to prevent staged-recipes growing any bigger than it needs to.

I guess it should be
```suggestion
  url: https://github.com/pypr/compyle/archive/v{{ version }}.tar.gz
```

---

The master branch is quite ahead of the v0.5 release(tag) on the github repo. That is why in the new meta.yaml file I have provided the source as a link to the master's tarball.
This is only temporary and we will make sure a more permanent sort of link to source is provided so that the tarball doesn't change on every commit on the master branch.

---

Conda-forge requires that releases be tagged, a small number of patches can be applied (see the conda-build docs) but these shouldn't affect the package's functionality. It is very confusing to install version 0.5 from conda-forge and find it is different from the "same" version installed elsewhere.

The best solution would be to ask upstream to tag a new version.